src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -184,4 +183,106 @@ public void Chat_007_banUser() {
     chatUserStaff.banUser(userToBeBanned);
     chatUserStaff.unBanUser(userToBeBanned);
   }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_008", "Chat", "Modals", "ChatTests"})
+  public void userBlockedCanNotCreatePrivateMessage() {
+    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+
+    switchToWindow(0);
+    chatUserOne.clickOnDifferentUser(userFive);
+    chatUserOne.selectPrivateMessageToUser(userFive);
+    chatUserOne.blockPrivateMessageFromUser(userFive);
+
+    switchToWindow(1);
+    chatWindow.refreshPage();
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.verifyBlockedUserIsUnableWritePrivateMessage(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_009", "Chat", "Modals", "ChatTests"})
+  public void userCanOpenMessageWall() {
+    openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.clickOpenUserMessageWall(userOne);
+    chatUserFive.switchToSecondTab();
+    chatWindow.refreshPage();
+    chatUserFive.verifyMessageWallOpened(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_010", "Chat", "Modals", "ChatTests"})
+  public void userCanOpenContributions() {
+    openChatForUser(userOne, userOnePassword);
+
+    switchToWindow(1);
+    SpecialVersionPage chatWindow = new SpecialVersionPage().open();
+    ChatPageObject chatUserFive = openChatForUser(userFive, userFivePassword);
+    chatUserFive.clickOnDifferentUser(userOne);
+    chatUserFive.clickOpenUserContributions(userOne);
+    chatUserFive.switchToSecondTab();
+    chatWindow.refreshPage();
+    chatUserFive.verifyUserContributionsOpened(userOne);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_011", "Chat", "Modals", "ChatTests"})
+  public void userCanNotBlockPrivateMessages() {
+    openChatForUser(userStaff, userStaffPassword);
+
+    switchToWindow(1);
+    new SpecialVersionPage().open();
+    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
+    chatUserOne.verifyUserCanNotBlockPrivateMessagesFromStaff(userStaff);
+  }
+
+  @DontRun(env = {"preview", "dev", "sandbox"})
+  @Test(groups = {"Chat_012", "Chat", "Modals", "ChatTests"})
+  public void userCanBeKickedOutFromChat() {
```

Comment:
is "kick" the name of the action as it appears in the functionality ?
If yes, let it stay. If it's "remove" or something like this, stick to the functionality naming